source/Calamari/TerraformCliExecutor.cs

```diff
@@ -205,17 +205,16 @@ Version GetVersion()
         {
             ExecuteCommandAndVerifySuccess(new[] { "--version" }, out string consoleOutput, true);
 
-            consoleOutput = consoleOutput.Replace("Terraform v", "");
-            int newLinePos = consoleOutput.IndexOf('\n'); // this is always \n in all OSes for unknown reasons
-            if (newLinePos != -1)
-                consoleOutput = consoleOutput.Substring(0, newLinePos);
-
-            if (!Version.TryParse(consoleOutput, out var version))
+            Version parsedVersion = null;
+            var versionString = Regex.Match(consoleOutput, @"Terraform v([0-9\.]*)");
```

Comment:
This only works because `consoleOuput` is a single line. I hope this is guaranteed behaviour.

Reply:
You can ask for the version in json - I wonder if this would be more reliable for us?

Reply:
```
PS C:\Users\andre> terraform version -json
{
  "terraform_version": "1.0.1",
  "platform": "windows_amd64",
  "provider_selections": {},
  "terraform_outdated": false
}
```

Reply:
Version `v0.11.15` doesn't support `--json` option. Still, I think it's best to first check json version and if that's not available fallback to regex.